prow/cmd/mkpod/main.go

```diff
@@ -74,7 +74,7 @@ func main() {
 	}
 
 	var job kube.ProwJob
-	if err := yaml.Unmarshal(rawJob, &job); err != nil {
+	if err := yaml.UnmarshalStrict(rawJob, &job); err != nil {
```

Comment:
```golang
type unmarshaller func([]byte, interface{}) error

var default = yaml.Unmarshal
```

Then in unit testing:
```golang
default = yaml.UnmarshalStrict
```
  

Or is the belief here that unmarshalstrict is the correct thing to do in cluster?

Reply:
mkpod is used similar-ishly to mkpj -- from developer workstations (or at least that was the original intent), so strict unmarshaling here would be validation prior to sending to the cluster ..?

Reply:
Sorry, the main thing I think should use non-strict is the one in config. Fine with any of the command-line utilities that create prow jobs using strict.